.github/workflows/ci.yml

```diff
@@ -67,7 +85,7 @@ jobs:
           echo "::set-env name=RUSTC_WRAPPER::sccache"
 
       - name: Test
-        run: cargo test -vv --locked --all-targets
+        run: cargo test -vv --locked --all-targets --target ${{ matrix.config.TARGET }}
```

Comment:
Does cargo automatically run the tests in qemu when the target != host?

Reply:
No. This relies on `binfmt_misc`(kernel module) which reads file headers and executes with the correct interpreter. Works sorta like a shebang.